pkg/api/controller_test.go

```diff
@@ -1341,22 +1342,22 @@ func TestController_SetupLakeFSHandler(t *testing.T) {
 		{
 			name: "accessKeyAndSecret",
 			key: &api.AccessKeyCredentials{
-				AccessKeyId:     "IKEAsneakers",
-				SecretAccessKey: "cetec astronomy",
+				AccessKeyId:     validAccessKeyID,
+				SecretAccessKey: "secret",
```

Comment:
Not a fan of [_Sneakers_](https://www.imdb.com/title/tt0105435/)??!?  Because that movie had [Len Adleman](https://adleman.usc.edu/) as a maths consultant...

Reply:
reverting back - it had space, at that point in time it wasn't valid.